src/index.css

```diff
@@ -71,7 +71,7 @@
   border: 0;
   border-top-left-radius: 18px;
   border-bottom-left-radius: 18px;
-  color: grey;
+  /* color: grey; */
```

Comment:
No need to keep comments in the code. You always review them from your commit history.
```suggestion
```